aiven/client/cli.py

```diff
@@ -6176,6 +6176,104 @@ def service__alloydbomni__google_cloud_private_key__delete(self) -> None:
             layout = ["client_email", "private_key_id"]
             self.print_response(output, json=self.args.json, table_layout=layout)
 
+    @arg.project
+    @arg.service_name
+    @arg(
+        "--operation",
+        help="Operation that is being allowed or denied.",
+        required=True,
+        choices=[
+            "Describe",
+            "DescribeConfigs",
+            "Alter",
+            "IdempotentWrite",
+            "Read",
+            "Delete",
+            "Create",
+            "ClusterAction",
+            "All",
+            "Write",
+            "AlterConfigs",
+            "CreateTokens",
+            "DescribeTokens",
+        ],
+    )
+    @arg(
+        "--resource-name",
+        help=(
+            "The resource to which ACLs should be added, when using LITERAL resource pattern type, "
+            "a name of '*' matches all resources of the selected type"
+        ),
+        required=True,
+    )
+    @arg(
+        "--resource-type",
+        help="Topic resource type to which ACLs should be added",
+        required=False,
+        choices=["Any", "Topic", "Group", "Cluster", "TransactionalId", "DelegationToken"],
+    )
```

Comment:
Just for user convenience, should we model command line args more similar to kafka-acls.sh?

It would mean that instead of simply giving resource-name as parameter, usage would be more like
`[--cluster | --group group | --topic topic | --transactional-id transactional-id ] `
where only one of these is given.

Also for cluster type name must be always `kafka-cluster`.

Also we could skip DelegationToken here because there is currently no support for those in aiven services.

Reply:
This is argparse based so is it possible to add mutually exclusive group as in https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group ?